lib/internal/streams/from.js

```diff
@@ -55,7 +55,7 @@ function from(Readable, iterable, opts) {
     } catch (e) {
       error = error || e;
     } finally {
-      cb(error);
+      process.nextTick(() => cb(error));
```

Comment:
If you do `process.nextTick(cb, error)` you don't allocate an unnecessary closure. It does the same thing.

Reply:
fixed